pytket/extensions/qiskit/backends/ibm_utils.py

```diff
@@ -173,6 +173,9 @@ def lightsabre(
         c: Circuit = qiskit_to_tk(applied_c)
         c.remove_blank_wires()
         c.rename_units({q: Node(q.index[0]) for q in c.qubits})
+        # Decompose CircBoxes corresponding to "If" and "Else" blocks in
+        #  conditional gates (qiskit IfElseOp).
+        DecomposeBoxes(excluded_types={OpType.BRIDGE}).apply(c)
```

Comment:
This resolves the remote test failure here -> https://github.com/CQCL/pytket-qiskit/actions/runs/14597476884/job/40946972788

Reply:
`BRIDGE` isn't a box type, so I don't see how this can have any effect.

Reply:
I'm aware of that but was confused by the error message. I've removed for now. See [432c707](https://github.com/CQCL/pytket-qiskit/pull/475/commits/432c707dce076adc6c2b16f7ca6b0bf0c86e90d4)